content/features.yaml

```diff
@@ -1,20 +1,16 @@
--
-  name: design
-  title: Simple, Clean and Modern Design.
-  body: Lorem ipsum dolor sit amet, ea eum labitur scripserit, illum complectitur deterruisset at pro. Odio quaeque reformidans est eu, expetendis intellegebat has ut, viderer invenire ut his. Has molestie percipit an. Falli volumus efficiantur sed id, ad vel noster propriae. Ius ut etiam vivendo, graeci iudicabit constituto at mea. No soleat fabulas prodesset vel, ut quo solum dicunt. Nec et amet vidisse mentitum. Cibo mutat nulla ei eam.
+- name: interactive
+  title: Interactive, collaborative learning spaces
+  body: Encapsulated learning activities in Activities or Inquiry Learning Spaces (ILS) and can be composed of various elements such as text, images, videos, and even more complex and specific technologies such as apps or labs. All these elements are finally presented in a dynamic and attractive layout and can be shared with a set of students or a wider audience.
   image: 'images/feature-image-1.png'
--
-  name: responsive
-  title: Fully Responsive.
-  body: Aenean condimentum, lacus sit amet luctus lobortis, dolores et quas molestias excepturi enim tellus ultrices elit, amet consequat enim elit noneas sit amet luctu. Quis nostrum exercitationem ullam corporis suscipit laboriosam.Our library is continually refreshed with the latest on web technology so you'll never fall behind. Quis nostrum exercitationem ullam corporis suscipit laboriosam.
+- name: offline
+  title: Offline Support
+  body: Graasp Desktop works while online or offline. Some functionalities such as Visit a Space, Sync and Spaces Nearby require an Internet connection.
```

Comment:
Graasp Desktop works both online and offline. While online, you can visit and download learning spaces using a link or by searching for spaces that have been tagged near your location. You can then use and share downloaded spaces, even if you no longer have access to an internet connection.